lib/model/item.dart

```diff
@@ -0,0 +1,12 @@
+/// Item:
+///   - objects of this class store a single item
+///   in a list (needs to be wrapped in an ItemWidget
+///   to be displayed)
```

Comment:
```suggestion
///     in a list (needs to be wrapped in an ItemWidget
///     to be displayed)
```
It'll look better if the beginning of these lines line up with line #2.